### DISCUSSION TITLE
DashboardLinks: Support variable expression in to tooltip

### DISCUSSION THREAD

**Discussion by huynhsamha** [grafana/grafana#38203] [Dashboards, Panels & Visualizations]
**What would you like to be added**:

Support inject variable to tooltip in dashboard link

Current situation (Grafana version **7.3.7**)

![c860dc369ae26abc33f3](https://user-images.githubusercontent.com/24195776/105121765-74f45380-5b07-11eb-943b-68fbaceb91b5.jpg)

In the link, variables are injected, but when I hover the link to show the tooltip, variables not work.
The place I inject variable to the tooltip is in the picture

![68e64cc10f15ff4ba604](https://user-images.githubusercontent.com/24195776/105121990-f4822280-5b07-11eb-9704-46aba4042cf1.jpg)

**Why is this needed**:

I think sometimes we want to show the detailed information in the tooltip when a user hovers a link, for example, an app name.

I hope this is a new feature in the next release.

Many thanks to Grafana, an awesome tool.

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
